phpBB/phpbb/mention/controller/user_mention.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\mention\controller;
+
+use phpbb\db\driver\driver_interface;
+use phpbb\request\request_interface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use phpbb\mention\helper\mention_helper;
+
+/**
+ * phpBB mentions main controller.
+ */
+class user_mention
+{
+	/**
+	* @var driver_interface
+	*/
+	private $db;
+
+	/**
+	* @var request_interface
+	*/
+	private $request;
+
+	/**
+	* @var mention_helper
+	*/
+	private $helper;
+
+	/**
+	* User Mention Controller Constructor
+	*
+	* @param $db          	\phpbb\db\driver\driver_interface
+	* @param $request       \phpbb\request\request_interface
+	*  @param $helper   	\phpbb\mention\helper\mention_helper
```

Comment:
Spacing.

Also, phpBB has the type before the variable, see https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/cron/manager.php#L52 for example. You should check all your docblocks, and make sure they are all the same.